include/SFML/Graphics/StencilMode.hpp

```diff
@@ -0,0 +1,214 @@
+////////////////////////////////////////////////////////////
+//
+// SFML - Simple and Fast Multimedia Library
+// Copyright (C) 2007-2022 Laurent Gomila (laurent@sfml-dev.org)
```

Comment:
```suggestion
// Copyright (C) 2007-2023 Laurent Gomila (laurent@sfml-dev.org)
```
Check the other copyright headers too.